VladimirPopov_WebForms/templates/webforms/fields/number.phtml

```diff
@@ -2,32 +2,32 @@
 <div class="input">
     <input
         type='number'
-        name='<?= $this->getFieldName() ?>'
-        id='<?= $this->getFieldId() ?>'
-        class='input__field <?= $this->getFieldClass() ?>'
-        style='<?= $this->getFieldStyle() ?>'
-        value="<?= htmlspecialchars($this->getFieldValue()); ?>"
-        placeholder="<?= $hint ?>"
+        name='<?= $block->escapeHtmlAttr($this->getFieldName()) ?>'
+        id='<?= $block->escapeHtmlAttr($this->getFieldId()) ?>'
+        class='input__field <?= $block->escapeHtmlAttr($this->getFieldClass()) ?>'
+        style='<?= $block->escapeHtmlAttr($this->getFieldStyle()) ?>'
+        value="<?= /* @noEscape */ htmlspecialchars($this->getFieldValue()); ?>"
```

Comment:
Everywhere, where is `htmlspecialchars`, I added `/* @noEscape */ ` but probably we should change  `htmlspecialchars` to `escapeHtmlAttr` and remove `/* @noEscape */`.
 (message from debbuger: `The use of function htmlspecialchars() is discouraged; use
    |         |     \Magento\Framework\Escaper->escapeHtml() instead`). 
Not sure if it should be done in this this or separate task.

Reply:
Yes you are right. I will ty to reach the author of extension first and then later we can update this extension templates.